index.html

```diff
@@ -450,13 +450,13 @@ <h3>Abstract Roles</h3>
 		</section>
 		<section id="requiredState">
 			<h3>Required States and Properties</h3>
-			<p><a>States</a> and <a>properties</a> specifically required for the <a>role</a> and subclass roles. Content authors MUST provide a non-empty <span>value</span> for required states and properties. Content authors MUST NOT use the value <code>undefined</code> for required states and properties, unless <code>undefined</code> is an explicitly-supported value of that state or property.</p>
+			<p><a>States</a> and <a>properties</a> specifically required for the <a>role</a> and subclass roles. Content authors MUST provide a non-empty <span>value</span> for required states and properties, except if current value is the default for that state or property for the role. Content authors MUST NOT use the value <code>undefined</code> for required states and properties, unless <code>undefined</code> is an explicitly-supported value of that state or property.</p>
```

Comment:
There should be no required states and properties with default values so we don't need this addition. (although saying that I see we missed one in our sweep of these which needs fixing https://github.com/w3c/aria/issues/700 )

Reply:
```suggestion
			<p><a>States</a> and <a>properties</a> specifically required for the <a>role</a> and subclass roles. Content authors MUST provide a non-empty <span>value</span> for required states and properties. Content authors MUST NOT use the value <code>undefined</code> for required states and properties, unless <code>undefined</code> is an explicitly-supported value of that state or property.</p>
```